File: e2e-tests/spring-examples/src/test/java/com/foo/rest/examples/spring/db/directintwithsql/DbDirectIntWithSqlController.java
package com.foo.rest.examples.spring.db.directintwithsql;

import com.foo.rest.examples.spring.db.directint.DbDirectIntController;
import org.evomaster.clientJava.controller.problem.ProblemInfo;
import org.evomaster.clientJava.controller.problem.RestProblem;

import java.util.Arrays;
import java.util.List;

public class DbDirectIntWithSqlController extends DbDirectIntController {

    @Override
    public ProblemInfo getProblemInfo() {
        return new RestProblem(
                "http://localhost:" + getSutPort() + "/v2/api-docs",
                //avoid using POST that would create the data
                Arrays.asList("/api/db/directint")
        );
    }
}

File: core-tests/e2e-tests/spring/spring-graphql/src/test/kotlin/com/foo/graphql/db/directintwithsql/DbDirectIntWithSQLController.kt
package com.foo.graphql.db.directintwithsql

import com.foo.graphql.db.SpringWithDbController
import com.foo.graphql.db.directint.DbDirectIntApplication


class DbDirectIntWithSQLController : SpringWithDbController(DbDirectIntApplication::class.java) {

    override fun schemaName() = "dbdirectIntQuery.graphqls"

}